admin/controllers/categories/views/edit/model.php

```diff
@@ -62,7 +62,8 @@
 			// forms are valid, save info
 			$saved = $cat->save($required_details_form, $custom_fields_form);
 			if ($saved) {
-				CMS::Instance()->queue_message('Category saved','success',Config::uripath() . "/admin/categories");
+				$msg = 'Category <a href="' . Config::uripath() . '/admin/categories/edit/' . $cat->id . '">' . $cat->title . '</a> ' . ($new_cat ? 'created' : 'updated');
```

Comment:
is there a reason not to use templating?

Reply:
fixed in b772c9972b1b6528cd219293d837976ffad29f09